modules/core/src/main/java/org/apache/ignite/internal/IgniteKernal.java

```diff
@@ -2142,9 +2142,9 @@ private void ackStart(RuntimeMXBean rtBean) {
             log.info(str);
         }
 
-        if (!ctx.state().clusterState().active()) {
-            U.quietAndInfo(log, ">>> Ignite cluster is not active (limited functionality available). " +
-                "Use control.(sh|bat) script or IgniteCluster interface to activate.");
+        if (!ClusterState.active(ctx.state().clusterState().state())) {
+            U.quietAndInfo(log, ">>> Ignite cluster is in INACTIVE state (limited functionality available). " +
+                "Use control.(sh|bat) script or IgniteCluster.state(ClusterState.ACTIVE) to change the state.");
```

Comment:
There's also ACTIVE_READ_ONLY state. Don't you think that we should also mention it here?